src/enzo/Zeus_yTransport_FDM.C

```diff
@@ -0,0 +1,286 @@
+/***********************************************************************
```

Comment:
as with other solvers, there are lots of commented-out lines that can be removed.

Reply:
Removed.